python/alibiexplainer.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.9-slim-bullseye
```

Comment:
what's the difference between `3.9-slim-bullseye` and `3.9-slim` ?

Reply:
whilst not being able to see differences between the builds of the 2 images ([3.9-slim](https://hub.docker.com/layers/python/library/python/3.9-slim/images/sha256-4169ae884e9e7d9bd6d005d82fc8682e7d34b7b962ee7c2ad59c42480657cb1d?context=explore) and [3.9-slim-bullseye](https://hub.docker.com/layers/python/library/python/3.9-slim-bullseye/images/sha256-4169ae884e9e7d9bd6d005d82fc8682e7d34b7b962ee7c2ad59c42480657cb1d?context=explore))

when built with  `python:3.9-slim-bullseye` the image reports 3 less critical CVEs:
<img width="661" alt="image" src="https://user-images.githubusercontent.com/35975404/175493717-427ea601-51e3-49a3-8dbf-b7d027af5170.png">

<img width="710" alt="image" src="https://user-images.githubusercontent.com/35975404/175493767-018688fb-75ee-42c2-80f7-3e729a4ff83e.png">